src/routes.ts

```diff
@@ -66,7 +66,7 @@ type OverridesProps =
   'postData' |
   'headers'
 
-const CHALLENGE_SELECTORS = ['.ray_id', '.attack-box']
+const CHALLENGE_SELECTORS = ['.ray_id', '.attack-box', '#trk_jschal_js']
```

Comment:
Since, according to you this selector seems to be present in almost all challenges, I suggest we look for it first. I think that would speed things up every so slightly.

```suggestion
const CHALLENGE_SELECTORS = ['#trk_jschal_js', '.ray_id', '.attack-box']
```